back/engines/commercial/multi_tenancy/db/seeds/community_monitor.rb

```diff
@@ -5,23 +5,34 @@
 module MultiTenancy
   module Seeds
     class CommunityMonitor < Base
+      def initialize(runner:, num_quarters: 2, ai_responses: false, locale: nil)
+        @num_quarters = num_quarters
+        @ai_responses = ai_responses
+        @locale = locale ||
+                  Locale.monolingual&.to_s ||
+                  AppConfiguration.instance.settings('core', 'locales').first ||
+                  I18n.default_locale
+        super(runner: runner)
+      end
+
       def run
-        @project = CommunityMonitorService.new.create_and_set_project
+        service = CommunityMonitorService.new
+        @project = service.project || service.create_and_set_project
         @phase = @project.phases.first
-        @question_keys = @phase.custom_form.custom_fields.reject(&:page?).pluck(:key)
-
-        # Responses for this quarter
-        runner.num_ideas.times { create_survey_response(Time.zone.now) }
-
-        # Responses for previous quarter
-        runner.num_ideas.times { create_survey_response(3.months.ago) }
+        @fields = @phase.custom_form.custom_fields.reject(&:page?)
```

Comment:
This makes me wonder again if pages should have been modeled as custom fields. I think I would have had a preference for modeling a form as many pages and each page having many custom fields.

Reply:
Yes that is interesting. We've also added categories to fields for community monitor because pages weren't quite right for that. Happy to talk about this another time